### ISSUE TITLE
Add Windows10/11-like theme

### ISSUE THREAD

**Issue by Matszwe02** [mortenn/BrowserPicker#96]
Will also include:
- [#70](https://github.com/mortenn/BrowserPicker/issues/70#issue-1180288327)
- [#51](https://github.com/mortenn/BrowserPicker/issues/51#issue-988183003)

I like the browser picker, but I don't like the windows7-like style - the whole ui looks very outdated. Maybe redesign the ui or make it selectable?

**Comment by mortenn** [comment]
The problem with this is I can't just flip a switch and make a WPF app use Windows 11 chrome, microsoft hasn't built WPF with support for that, so I would need to either get some chrome package that does this and will be maintained, or hand roll all the style.

The suggested path is to replace WPF with WinUI, which I guess I will have to investigate at some point. that sounds like a 3.0 thing, though.

**Comment by Matszwe02** [comment]
> The problem with this is I can't just flip a switch and make a WPF app use Windows 11 chrome, microsoft hasn't built WPF with support for that, so I would need to either get some chrome package that does this and will be maintained, or hand roll all the style.
> 
> The suggested path is to replace WPF with WinUI, which I guess I will have to investigate at some point. that sounds like a 3.0 thing, though.

Thanks for the response. However I think the easiest way will be to manually change the theme to match the whole "windows style", it's not a complicated app, so there won't be much to change I think.

**Comment by mortenn** [comment]
yes, it's not a big app, but I don't really have the spare time to do that, and I'm not a wpf frontend designer :3
I'll be more than happy to take a PR for this, though, chrome is nice.

**Comment by mipa87** [comment]
Working on something 🙂

![Image](https://github.com/user-attachments/assets/aacc0a89-39dc-4566-8af9-b01ba58c8efc)

![Image](https://github.com/user-attachments/assets/2ad7ad12-0898-4c36-b5ac-d85755ab5fe6)

![Image](https://github.com/user-attachments/assets/3a49a644-fa75-4b72-9da8-3c5763f4f7b6)
